projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/index.js

```diff
@@ -0,0 +1,94 @@
+/* global launchpadModalOptions */
+
+import { Modal, Button, CheckboxControl } from '@wordpress/components';
+import { usePrevious } from '@wordpress/compose';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { __ } from '@wordpress/i18n';
+import { useEffect, useState } from 'react';
+import './editor.scss';
+
+export const name = 'launchpad-save-modal';
+
+export const settings = {
+	render: function LaunchpadSaveModal() {
+		const isSaving = useSelect(
+			select => select( editorStore ).isSavingNonPostEntityChanges(),
+			[]
+		);
+		const prevIsSaving = usePrevious( isSaving );
+		const [ isModalOpen, setIsModalOpen ] = useState( false );
+		const [ dontShowAgain, setDontShowAgain ] = useState( false );
+		const [ isChecked, setIsChecked ] = useState( false );
+		const [ , siteSlug ] = launchpadModalOptions.siteUrlOption.split( '//' );
+		const isInsideSiteEditor = window.location.href.includes( 'site-editor' );
+
+		useEffect( () => {
+			if ( prevIsSaving === true && isSaving === false ) {
+				setIsModalOpen( true );
+			}
+		}, [ isSaving, prevIsSaving ] );
+
+		const showModal =
+			launchpadModalOptions.launchpadScreenOption === 'full' &&
+			launchpadModalOptions.siteIntentOption === 'link-in-bio' &&
```

Comment:
Would it make sense to enable this modal for the Free Flow now as well?

Reply:
Yes! But I think it would be easier to address this in a follow up PR since we'll need different content inside the modal